### DISCUSSION TITLE
Windows App SDK 1.6.2 (1.6.241106002)

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#4879] [Announcements]
## Windows App SDK 1.6.2 (1.6.241106002)

Windows App SDK 1.6.2 is a stable servicing release that fixes critical bugs for 1.6.

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using the Windows App SDK 1.6 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

Check out our [release notes](https://aka.ms/windowsappsdk/1.6/1.6.2/release-notes) for the list of issues fixed in this release.

### Updating to this version

The 1.6.2 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.6.241106002

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.6.2'>Windows App SDK 1.6.2 (1.6.241106002)</a>.</em>

**Comment by Balkoth** [comment]
I think it is pretty sad that styles are broken on Windows 10 since 1.6.0 and that this bug is still not fixed: https://github.com/microsoft/microsoft-ui-xaml/issues/10014

**Comment by danielgruethling** [comment]
Any info why it was unlisted again?
![grafik](https://github.com/user-attachments/assets/032709d4-c499-4eee-a3c6-5de01c10192f)

**Reply by ghost1372** [reply]
i created an issue
https://github.com/microsoft/WindowsAppSDK/issues/4887
